eventcore_kafka/producer.py

```diff
@@ -10,19 +11,40 @@ class KafkaProducer(Producer):
     :param servers: list of brokers to consume from.
     """
 
-    def __init__(self, servers):
+    source = ''
+
+    def __init__(self, servers, source=None):
```

Comment:
had no time to change this but technically you just want too initiate this class with a 'config' where the servers and source is part of that config. Made an issue for that: https://github.com/maikelboogerd/eventcore-kafka/issues/4